### PR TITLE
Fix/pg login

### PR DESCRIPTION
**Pull request by erenkan**

### CODE REVIEW

File: themes/mCommerceTheme/styles/components/FlHomeCategory.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+				"font": {
+					"size": 20,
+					"bold": true,
+					"italic": false,
+					"family": "Nunito",
+					"style": "Bold"
+				},
+				"marginLeft": 10,
+				"testId": "NrZ-2VxaH",
+				"textColor": "rgba( 62, 66, 63, 1 )"
```
Comment (alimoncul) [inline themes/mCommerceTheme/styles/components/FlHomeCategory.json]: variables :(

File: themes/mCommerceTheme/styles/components/FlHomeCategory.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+				},
+				"height": 80,
+				"testId": "-ICBM385d",
+				"width": null
+			},
+			"&-lblCategoryName": {
+				"flexProps": {
+					"flexGrow": 2
+				},
+				"font": {
```
Comment (alimoncul) [inline themes/mCommerceTheme/styles/components/FlHomeCategory.json]: move this font to textstylecatalog

**Review by alimoncul — CHANGES_REQUESTED** [review]

File: themes/mCommerceTheme/styles/components/FlHomeCategory.json (hunk 1 above)
Comment (erenkan) [reply themes/mCommerceTheme/styles/components/FlHomeCategory.json]: done

File: themes/mCommerceTheme/styles/components/FlHomeCategory.json (hunk 2 above)
Comment (erenkan) [reply themes/mCommerceTheme/styles/components/FlHomeCategory.json]: done

**Pull request merged**